### PR TITLE
#166880007 Enable follow or unfollow authors 

### PR DESCRIPTION
**Pull request by Rhytah**
#### What does this PR do?
- Enable follow or unfollow authors  

#### Description of Task to be completed?

- [x] Configure following an author User Interface

- [x] Configure unfollowing an author User Interface

#### Any background context you want to provide?
Only authenticated users should be able to follow and or unfollow other users.

#### What are the relevant pivotal tracker stories?
[#166880007](https://www.pivotaltracker.com/story/show/166880007)

### CODE REVIEW

File: src/actions/profile/followActions.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  console.log('it reaches here');
+  return axios.get(`https://ah-backend-kronos-staging.herokuapp.com/api/profiles/${username}/followers`, {
+    headers: {
+      'content-type': 'application/json',
+      Authorization: `Bearer ${sessionStorage.getItem('token')}`,
+    },
+  })
+    .then((response) => {
+      console.log(response, 'Rhytah');
+      dispatch({
```
Comment (reifred) [inline src/actions/profile/followActions.js]: Console.log should be removed.

File: src/actions/profile/followActions.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        type: profileActionTypes.FETCH_FOLLOWING_PROFILES,
+        payload: response.data.followers,
+      });
+      console.log(response, 'fetch follower usernamepo[[/////////////');
+    });
```
Comment (reifred) [inline src/actions/profile/followActions.js:82]: Still, at this point remove the console.log

File: src/components/articles/articleComponent.jsx [hunk 3]
```diff
@@ -232,6 +270,12 @@ Article.propTypes = {
   getSingleArticle: PropTypes.func.isRequired,
   article: PropTypes.shape({}),
   isFetching: PropTypes.bool.isRequired,
+  // followAuthorAction: PropTypes.func,
+  // fetchedArticle: PropTypes.shape({
+  //   author: PropTypes.shape({
+  //     username: PropTypes.string.isRequired,
+  //   }).isRequired,
+  // }),
```
Comment (reifred) [inline src/components/articles/articleComponent.jsx]: Kindly remove this commented code.

**Review by reifred — CHANGES_REQUESTED** [review]
Thanks @Rhytah for the good work done, however kindly remove the commented code and console.logs in various files of your files mentioned below and those not mentioned yet.

File: src/components/articles/articleComponent.jsx (hunk 3 above)
Comment (Rhytah) [reply src/components/articles/articleComponent.jsx]: Thanks, @reifred, let me clear this

**Review by Manorlds-Eaglespark — APPROVED** [review]
This is good work. The feature works as it should. The look can be improved.

**Review by reifred — APPROVED** [review]
Thanks @Rhytah, working on the requested changes. The feature is working fine as expected.

**Comment by Rhytah** [comment]
> This is good work. The feature works as it should. The look can be improved.

@Manorlds-Eaglespark thanks for the feedback, will have this worked upon in the next product iteration. We can share thoughts and come up with a better design.

**Review by kcharles52 — APPROVED** [review]

**Pull request merged**